### DISCUSSION TITLE
per capita consumption rate too low?

### DISCUSSION THREAD

**Discussion by vtrijoulet** [NOAA-EDAB/ms-keyrun#81] [Q&A]
Am I understanding well that the per capita consumption rate is in simPerCapCons, variable "intakeg"?
It seems very low, for example a 20 year old cod (agecl=10) eats on average (taken across years) 125 g of food per year. 
When I multiply "totNagecl" by "intakeg" from the same table I do get "totconsagecl" in tons so it seems that the numbers all add up but the consumption rate values are a bit unrealistic. Is there a problem of unit in this table?

**Comment by sgaichas** [comment]
I agree that this seems low compared with data from 'real' systems. I did some comparisons [here](https://sgaichas.github.io/poseidon-dev/PerCapitaConsumption.html) between two Atlantis models and a couple of different approaches to get per capita consumption. The current approach (Approach 2 in the linked doc) is better than the first approach but perhaps still low compared with what we would expect based on real data. I am hoping that it is at least internally consistent within Atlantis but this is worth discussing further.

**Comment by gustavdelius** [comment]
Is consumption rate at age something that is actually observable in the real world? If not, should it be included in the data that we use to calibrate our models? If yes, what about consumption rate at size, which is what size-based models would use? Atlantis could presumably export that as well.

**Comment by vbartolino** [comment]
> per capita consumption [...] perhaps still low [...] I am hoping that it is at least internally consistent within Atlantis but this is worth discussing further.

Even if internally consistent, having simulations with a low predators' consumption could be problematic as it will result in weak predator-prey interactions, hence implicitly diminishing the usefulness and justification for using multispecies models. Can this be checked and eventually corrected in the simulations?

**Comment by vbartolino** [comment]
from an overview of the food intake simulated data, I'm confused by some of the patterns. For instance, for some species the food intake per capita is larger in younger ages, how can this be possible? Moreover, for some species the range of per capita consumption is unexpectedly too small
 
```ggplot(mskeyrun::simPerCapCons %>% filter(variable=="intakeg"), aes(as.factor(agecl),value)) + ```
```   geom_boxplot() + ```
```  facet_wrap(~Name, scale="free")```

**Comment by sgaichas** [comment]
I agree, this still seems off and I will double check this as I am testing new functions in atlantisom. I may be assuming something about conversions that is incorrect, as I just learned for biomass pools when making the Rpath input dataset.
